### DISCUSSION TITLE
네이버 백신예약 성공하신 분 계신가요?

### DISCUSSION THREAD

**Discussion by HappyHand2** [kimytsc/covid-rest-vaccine-macro#36] [Q&A] [answered]
몇 시간 째 돌리고 있는데 계속 예약 실패가 뜨네요ㅠㅠ
시간이랑 병원명은 계속 업데이트 되는것으로 보아 프로그램은 잘 돌아가고 있는거 같습니다.
제가 전산상으로 잔여백신 예약이 안되는 상태였는데 며칠전에 질병청 통해서 다시 잔여백신 예약이 되도록 조치를 받은 상태입니다.
그래도 잔여백신 예약이 가능하도록 바뀐지 확인할 방법이 없어서 계속 실패하는게 제 문제인지 매크로가 못 잡는건지 확인하고 싶어서 여쭤봅니다~

**Comment by HappyHand2** [comment, marked as answer]
잘 됐네요! 정상작동하네요 감사합니다~

**Reply by worud47** [reply]
예약성공하신건가요?

**Reply by HappyHand2** [reply]
네 성공했습니다
